scripts/Dockerfile

```diff
@@ -19,20 +11,25 @@ RUN curl -L https://golang.org/dl/go${GO_VERSION}.linux-amd64.tar.gz -o /tmp/go.
     && go version \
     && rm -f /tmp/go.tar.gz
 
-# test user
-ARG TEST_USER
-ENV TEST_USER=${TEST_USER:-test}
+# Install protoc
+RUN PROTOC_ZIP=protoc-3.19.4-linux-x86_64.zip \
```

Comment:
It would be great to have constants for protoc and protoc-gen-grpc-gateway versions (3.19.4 and 2.8.0)

Reply:
Moved versions to ENV variables